### PR TITLE
Fix the Bang.

### LINKED ISSUE #14479: [4.0] [3.7] Control Panel: Joomla! vs Joomla
### Steps to reproduce the issue
Load Control Panel,Message at checking Joomla:
bildschirmfoto 2017-03-11 um 08 00 21
After Check:
bildschirmfoto 2017-03-11 um 08 00 34
### Expected result
Same Wording.
### System information (as much as possible)
Joomla! 4.0.0-dev
macOS Sierra, 10.12.3
Firefox 52.0 (64-bit)
PHP 7.0.15
#### MAMP 4.1.1
- PHP 7.0.15
- MySQLi 5.6.35

### PR DESCRIPTION
**Pull request by tonypartridge**
Fixes #14479 
Theres no need for a bang according to the No-Bang! documentation here:
https://developer.joomla.org/en-gb-user-interface-text-guidelines/joomla-name.html

### CODE REVIEW

File: administrator/language/en-GB/en-GB.plg_quickicon_joomlaupdate.ini [hunk 1]
```diff
@@ -3,13 +3,13 @@
 ; License GNU General Public License version 2 or later; see LICENSE.txt, see LICENSE.php
 ; Note : All ini files need to be saved as UTF-8
 
-PLG_QUICKICON_JOOMLAUPDATE="Quick Icon - Joomla! Update Notification"
-PLG_QUICKICON_JOOMLAUPDATE_CHECKING="Checking Joomla! ..."
-PLG_QUICKICON_JOOMLAUPDATE_ERROR="Unknown Joomla! ..."
+PLG_QUICKICON_JOOMLAUPDATE="Quick Icon - Joomla Update Notification"
+PLG_QUICKICON_JOOMLAUPDATE_CHECKING="Checking Joomla ..."
```
Comment (C-Lodder) [inline administrator/language/en-GB/en-GB.plg_quickicon_joomlaupdate.ini]: Could just use `Checking Joomla&hellip;`

File: administrator/language/en-GB/en-GB.plg_quickicon_joomlaupdate.ini [hunk 2]
```diff
@@ -3,13 +3,13 @@
...
+PLG_QUICKICON_JOOMLAUPDATE_ERROR="Unknown Joomla ..."
```
Comment (C-Lodder) [inline administrator/language/en-GB/en-GB.plg_quickicon_joomlaupdate.ini]: Could just use `Unknown Joomla&hellip;`

File: administrator/language/en-GB/en-GB.plg_quickicon_joomlaupdate.ini [hunk 3]
```diff
@@ -3,13 +3,13 @@
...
 PLG_QUICKICON_JOOMLAUPDATE_GROUP_DESC="The group of this plugin (this value is compared with the group value used in <strong>Quick Icons</strong> modules to inject icons)."
 PLG_QUICKICON_JOOMLAUPDATE_GROUP_LABEL="Group"
-PLG_QUICKICON_JOOMLAUPDATE_UPDATEFOUND="Joomla! <span class='label label-important'>%s</span>, Update now!"
+PLG_QUICKICON_JOOMLAUPDATE_UPDATEFOUND="Joomla <span class='label label-important'>%s</span>, Update now!"
```
Comment (C-Lodder) [inline administrator/language/en-GB/en-GB.plg_quickicon_joomlaupdate.ini]: Replace `label label-important` with `badge badge-important`

File: administrator/language/en-GB/en-GB.plg_quickicon_joomlaupdate.ini [hunk 4]
```diff
@@ -3,13 +3,13 @@
...
 PLG_QUICKICON_JOOMLAUPDATE_UPDATEFOUND_BUTTON="Update Now"
-PLG_QUICKICON_JOOMLAUPDATE_UPDATEFOUND_MESSAGE="Joomla! <span class='label label-important'>%s</span> is available:"
+PLG_QUICKICON_JOOMLAUPDATE_UPDATEFOUND_MESSAGE="Joomla <span class='label label-important'>%s</span> is available:"
```
Comment (C-Lodder) [inline administrator/language/en-GB/en-GB.plg_quickicon_joomlaupdate.ini]: Replace `label label-important` with `badge badge-danger`

File: administrator/language/en-GB/en-GB.plg_quickicon_joomlaupdate.ini (hunk 1 above)
Comment (brianteeman) [inline administrator/language/en-GB/en-GB.plg_quickicon_joomlaupdate.ini]: Not in favour of hellip as it makes it harder for translation

**Comment by franz-wohlkoenig** [comment]
I have tested this item :white_check_mark: successfully on 1de2f71207b943bec6518620484cd19ff0a89bd0<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/14728">issues.joomla.org/tracker/joomla-cms/14728</a>.</sub>

File: administrator/language/en-GB/en-GB.plg_quickicon_joomlaupdate.ini (hunk 1 above)
Comment (tonypartridge) [inline administrator/language/en-GB/en-GB.plg_quickicon_joomlaupdate.ini]: But can we get my pull tested/merged? Then discuss the &hellip; separately?

**Comment by tonypartridge** [comment]
Anyone got any more tests / comments on this? Or shall I just close it?

**Comment by C-Lodder** [comment]
I have tested this item :white_check_mark: successfully on e264f79fd5e8c6e07bae4b6dd9bc650d7b9cf5ac<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/14728">issues.joomla.org/tracker/joomla-cms/14728</a>.</sub>

**Comment by tonypartridge** [comment]
@C-Lodder I totally did :-)

**Comment by franz-wohlkoenig** [comment]
RTC as there are 2 successfully Tests?

**Comment by jeckodevelopment** [comment]
technically there are commits after the tests..

**Comment by C-Lodder** [comment]
I have tested this item :white_check_mark: successfully on b4fb1889eb902b7b4ac86120c0fe70da38ea6db7<br /><br />again...<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/14728">issues.joomla.org/tracker/joomla-cms/14728</a>.</sub>

**Pull request merged**